docs/cudf/source/user_guide/config.md

```diff
@@ -0,0 +1,8 @@
+# Options
```

Comment:
Rename the file to match Pandas (options). https://pandas.pydata.org/docs/user_guide/options.html